src/cleanlab_tlm/utils/chat_completions.py

```diff
@@ -0,0 +1,123 @@
+import asyncio
```

Comment:
my recommended tutorial to make for this is here:

https://github.com/cleanlab/cleanlab-tlm/pull/74#issuecomment-2961137748

would be good to investigate whether the better monkey-patch version of workflow 2 in that link can be made to work with this API